### PR TITLE
Silas/role specific questions2

### PR DESCRIPTION
**Pull request by silastsui**

### CODE REVIEW

**Comment by silastsui** [comment]
need to be careful of merge conflicts when merging mine and susan's branches

File: src/app/student-application/student-application.component.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h2 class="uppercase">Basic information</h2>
       <div class="col-sm-6 col-md-6 u-margin-bottom-25 u-padding-right-only-10"
            [ngClass]="{
             'invalid':
-              !studentApplicationForm.controls['firstName'].valid &&
+              !studentApplicationForm .controls['firstName'].valid &&
```
Comment (Ruxuan) [inline src/app/student-application/student-application.component.html]: Is this intentional?

File: src/app/student-application/student-application.component.html [hunk 2]
```diff
@@ -158,39 +158,56 @@ <h2 class="uppercase">Position preference</h2>
...
-      <small *ngIf="!studentApplicationForm.controls['positions'].valid &&
-                    studentApplicationForm.controls['positions'].touched">Required. Please select at least one option.</small>
     </div>
     <div class="row">
-      <h2 class="uppercase">Short answers</h2>
-      <p>Please answer the following questions in a few sentences.</p>
-    </div>
-    <div *ngFor="let q of shortAnswers; let i = index"
+      <h2 class="uppercase">Role Specific Questions</h2>
+      <p>Please answer the following questions in a few sentences. Note that responses will not be saved if you uncheck roles later. (max length: 567 characters)</p>
```
Comment (Ruxuan) [inline src/app/student-application/student-application.component.html]: Is it still 567 characters? Or was it changed to 789?

File: src/configs/application_config.ts [hunk 3]
```diff
@@ -97,29 +151,71 @@ export const APPLICATION_CONFIG = {
...
+        answer: ''
+      },
+      'Design Director': {
+        question: 'What’s your processes for working with other designers and developers?*',
+        placeholder: 'I create a Trello board',
+        answer: ''
+      },
+      'VP External': {
+        question: 'How do you know know whether a non-profit has a goal that resonates with Blueprint?*',
+        placeholder: 'The non-profit must be',
```
Comment (Ruxuan) [inline src/configs/application_config.ts]: 'The non-profit must be ...'

File: src/app/student-application/student-application.component.html (hunk 1 above)
Comment (chuqianli) [reply src/app/student-application/student-application.component.html]: doesn't seem like it. Remove!

Comment (silastsui) [reply src/app/student-application/student-application.component.html]: oof

File: src/app/student-application/student-application.component.html (hunk 2 above)
Comment (silastsui) [reply src/app/student-application/student-application.component.html]: you're right, it's 789

**Pull request merged**